### PR TITLE
Added four websites to KE's list.

### PR DESCRIPTION
**Pull request by Tafiti**

### CODE REVIEW

File: lists/ke.csv [hunk 1]
```diff
@@ -123,7 +123,7 @@ https://www.the-star.co.ke,NEWS,News Media,2016-06-13,OONI,
 http://www.the-star.co.ke/news/2015/12/11/ti-bribes-report-shows-kenya-police-judiciary-among-most-corrupt_c1259309,POLR,Political Criticism,2016-06-13,OONI,
 http://www.theeastafrican.co.ke,NEWS,News Media,2014-04-15,citizenlab,
 http://www.tikenya.org,POLR,Political Criticism,2014-04-15,citizenlab,
-"http://www.time.com/time/world/article/0,8599,2057966,00.html",HUMR,Human Rights Issues,2014-04-15,citizenlab,
```
Comment (sneft) [inline lists/ke.csv]: Since there are commas in the URL, it must be in quotation marks to avoid breaking the CSV

Comment (Tafiti) [inline lists/ke.csv]: Let me fix that. Thanks.

File: lists/ke.csv [hunk 2]
```diff
@@ -133,3 +133,7 @@ http://www.worldpoliticsreview.com/articles/17983/kenya-tries-to-stifle-any-crit
 http://www.wrapkenya.or.ke,HUMR,Human Rights Issues,2014-04-15,citizenlab,
 http://sportpesa.com/,GMB,Gambling,2017-03-20,CIPIT,
 https://sportpesa.com/,GMB,Gambling,2017-03-20,CIPIT,
+https://www.iebc.or.ke/,GOVT,Government,2017-05-25,CIPIT,
+https://jubileepamoja.co.ke/,MISC,Miscelaneous content,2017-05-25,CIPIT,
+https://rao.co.ke/,POLR,Political Criticism,2015-05-25,CIPIT,
+https://www.standardmedia.co.ke/article/2000229727/communications-authority-to-monitor-private-talk-and-texts-during-poll,NEWS,News Media,2017-05-25,CIPIT,
```
Comment (hellais) [inline lists/ke.csv]: Seems like this URL now redirects to the homepage.

Comment (Tafiti) [inline lists/ke.csv]: As noted earlier, it would be interesting to see if the media company will revive the url.

**Comment by hellais** [comment]
Ok I am going to proceed with merging this.

**Pull request merged**